resources/views/users/index.blade.php

```diff
@@ -3,6 +3,27 @@
 @section('content')
 
 <div class="container">
+
```

Comment:
can you create a separate template for it? so that we can use the same template in other places as well and don't need to create a separate template for list sender-identity?